vaadin-grid-flow-parent/vaadin-grid-flow/src/main/java/com/vaadin/flow/component/grid/Grid.java

```diff
@@ -3210,9 +3216,11 @@ private void setSortOrder(List<GridSortOrder<T>> order,
 
         sortOrder.clear();
         if (order.isEmpty()) {
-            // Grid is not sorted anymore.
+            // Grid is not sorted anymore with client-side sorting.
```

Comment:
This is a bit off in the sense that it is not "client-side" sorting, but column based sorting which could be applied from server side code too, not just by user clicking the column header(s).

Reply:
Comment modified.